pages/settings/teams.tsx

```diff
@@ -153,4 +166,16 @@ export default function Teams(props) {
       </SettingsShell>
     </Shell>
   );
-}
\ No newline at end of file
+}
+
+// Export the `session` prop to use sessions with Server Side Rendering
+export const getServerSideProps: GetServerSideProps<{session: Session | null}> = async (context) => {
+    const session = await getSession(context);
+    if (!session) {
+      return { redirect: { permanent: false, destination: '/auth/login' } };
```

Comment:
Codacy has a fix for the issue: [Replace `······return·{·redirect:·{·permanent:·false,·destination:·'/auth/login'` with `····return·{·redirect:·{·permanent:·false,·destination:·"/auth/login"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
    return { redirect: { permanent: false, destination: "/auth/login" } };
```